apps/web/src/common/modules/navigations/lnb/modules/LNBDividerMenuItem.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+import { PDivider } from '@spaceone/design-system';
+</script>
+
+<template>
+    <div class="l-n-b-divider-menu-item">
+        <p-divider />
+    </div>
+</template>
+
+<style lang="postcss" scoped>
+.l-n-b-divider-menu-item {
+    margin-top: 1.25rem;
+    margin-bottom: 1.25rem;
+    height: 0;
+}
+</style>
```

Comment:
Q. Is there a reason for separating components even though there's only one divider?

Reply:
There are some cases without clear usage, so I separated this for usage with spec.